benchmark/preconditioner/preconditioner.cpp

```diff
@@ -90,13 +90,33 @@ gko::precision_reduction parse_storage_optimization(const std::string &flag)
 const std::map<std::string, std::function<std::unique_ptr<gko::LinOpFactory>(
                                 std::shared_ptr<const gko::Executor> exec)>>
     precond_factory{
-        {"jacobi", [](std::shared_ptr<const gko::Executor> exec) {
+        {"jacobi",
+         [](std::shared_ptr<const gko::Executor> exec) {
              return gko::preconditioner::Jacobi<etype>::build()
                  .with_max_block_size(FLAGS_max_block_size)
                  .with_storage_optimization(
                      parse_storage_optimization(FLAGS_storage_optimization))
                  .with_accuracy(FLAGS_accuracy)
                  .on(exec);
+         }},
+        {"ilu",
```

Comment:
Maybe I am mistaken here, but isnt this actually a ParILU ? I guess this discussion should move out as I see that the class is [`named ILU`](https://github.com/ginkgo-project/ginkgo/blob/develop/include/ginkgo/core/preconditioner/ilu.hpp#L113) but the [documentation](https://github.com/ginkgo-project/ginkgo/blob/develop/include/ginkgo/core/preconditioner/ilu.hpp#L81) and [some parts](https://github.com/ginkgo-project/ginkgo/blob/develop/include/ginkgo/core/preconditioner/ilu.hpp#L112) mention ParILU